### ISSUE TITLE
`templaterepo` as URL

### ISSUE THREAD

**Issue by jcolomb** [G-Node/tonic#30]
the doc says : `templaterepo` should be of the form `user/repository`

If the form would be a url, one could use one single template repo for different gin instances, as long as the template repo is public.

**Comment by achilleas-k** [comment]
Good idea.  I'd have to make some small changes to accommodate this, but I don't see any issues with it on first consideration.

**Comment by jcolomb** [comment]
would be nice to get this done, also for the submodules by the way. would help with testing.
(the current template is on gin, I would need to redo one on each platform to test...)

**Comment by achilleas-k** [comment]
Submodules can be anywhere since they're fetched using the URL in `.gitmodules`.

One thing to point out: A URL for the repo would mean it would have to be public.  This would also need to be the case if it's on the same server and part of the org.

**Comment by jcolomb** [comment]
`.gitmodules` is using relative links now, so this may not be right anymore.
